### PR TITLE
Add com.amazon.aws.cloudfront/wd_access_log/jsonschema/1-0-6

### PR DESCRIPTION
**Pull request by BenFradet**

### CODE REVIEW

File: schemas/com.amazon.aws.cloudfront/wd_access_log/jsonschema/1-0-6 [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    },
+    "csProtocolVersion": {
+      "type": ["string", "null"],
+      "maxLength": 8
+    },
+    "fleStatus": {
+      "type": ["string", "null"],
+      "maxLength": 64
+    },
+    "fleEncryptedFields": {
```
Comment (chuwy) [inline schemas/com.amazon.aws.cloudfront/wd_access_log/jsonschema/1-0-6]: Shouldn't it be integer?  From [docs](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/AccessLogs.html) it looks like it is a natural number

Comment (BenFradet) [reply schemas/com.amazon.aws.cloudfront/wd_access_log/jsonschema/1-0-6]: problem is that it can be "-" too

**Comment by alexanderdean** [comment]
We are hoping to get R94 out today - can we get a green tick?

File: schemas/com.amazon.aws.cloudfront/wd_access_log/jsonschema/1-0-6 (hunk 1 above)
Comment (chuwy) [reply schemas/com.amazon.aws.cloudfront/wd_access_log/jsonschema/1-0-6]: Okay, makes sense.

**Comment by alexanderdean** [comment]
Merged

**Pull request closed without merging**